reports/hemang.md

```diff
@@ -179,7 +179,7 @@ activity:
  - 32:
    - repo: landing-frontend
    - link: https://github.com/coala/landing-frontend/commits/master?author=hemangsk
-   - details: > 
+   - details: >
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.